atom-client/src/test/java/org/motechproject/atomclient/it/AtomClientServiceBundleIT.java

```diff
@@ -0,0 +1,118 @@
+            throw e;
+        }
+    }
+
+
+    @Before
+    public void setupAtomServer() {
+
+        final String atomResponse =
+                "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
+                "<feed xmlns=\"http://www.w3.org/2005/Atom\">\n" +
+                "  <title>Patient AOP</title>\n" +
+                "  <link rel=\"self\" type=\"application/atom+xml\" href=\"http://192.168.33.10:8080/openmrs/ws/atomfeed/patient/recent\" />\n" +
+                "  <link rel=\"via\" type=\"application/atom+xml\" href=\"http://192.168.33.10:8080/openmrs/ws/atomfeed/patient/35\" />\n" +
+                "  <link rel=\"prev-archive\" type=\"application/atom+xml\" href=\"http://192.168.33.10:8080/openmrs/ws/atomfeed/patient/34\" />\n" +
+                "  <author>\n" +
+                "    <name>OpenMRS</name>\n" +
+                "  </author>\n" +
+                "  <id>bec795b1-3d17-451d-b43e-a094019f6984+35</id>\n" +
+                "  <generator uri=\"https://github.com/ICT4H/atomfeed\">OpenMRS Feed Publisher</generator>\n" +
+                "  <updated>2016-02-23T03:31:13Z</updated>\n" +
+                "  <entry>\n" +
+                "    <title>Patient</title>\n" +
+                "    <category term=\"patient\" />\n" +
+                "    <id>tag:atomfeed.ict4h.org:8b38f530-6c19-4e52-a3be-d3fce58132ce</id>\n" +
+                "    <updated>2016-02-23T03:27:59Z</updated>\n" +
+                "    <published>2016-02-23T03:27:59Z</published>\n" +
+                "    <content type=\"application/vnd.atomfeed+xml\"><![CDATA[/openmrs/ws/rest/v1/patient/aca97062-35c5-4a23-baf8-56e6eec76320?v=full]]></content>\n" +
+                "  </entry>\n" +
+                "  <entry>\n" +
+                "    <title>Patient</title>\n" +
+                "    <category term=\"patient\" />\n" +
+                "    <id>tag:atomfeed.ict4h.org:9cc95d86-1b8e-4582-a774-83cf4b73c23e</id>\n" +
+                "    <updated>2016-02-23T03:31:13Z</updated>\n" +
+                "    <published>2016-02-23T03:31:13Z</published>\n" +
+                "    <content type=\"application/vnd.atomfeed+xml\"><![CDATA[/openmrs/ws/rest/v1/patient/aca97062-35c5-4a23-baf8-56e6eec76320?v=full]]></content>\n" +
+                "  </entry>\n" +
+                "</feed>\n";
+
+        String httpServerURI = SimpleHttpServer.getInstance().start("foo", HttpStatus.SC_OK, atomResponse);
+        getLogger().debug("verifyServiceFunctional - We have a server listening at {}", httpServerURI);
+
+        configService.setFeedConfigs(new FeedConfigs(
+                new HashSet<>(Arrays.asList(
+                        new FeedConfig(httpServerURI, "/([0-9a-f-]*)\\?")
+                ))));
+        atomClientService.scheduleFetchJob("");
+    }
+
+
+    @Test
+    public void verifyService() {
+        assertNotNull(atomClientService);
+        atomClientService.fetch();
+        atomClientService.fetch();
+        atomClientService.fetch();
```

Comment:
Can this ever fail? It seems to only log something in case of problems. We should verify that it actually does something that we expect.